src/test/java/com/marklogic/appdeployer/DefaultAppConfigFactoryTest.java

```diff
@@ -929,6 +929,70 @@ void globalKeyStoreAndTrustStore() {
 		assertEquals("SunX5092", config.getAppServicesTrustStoreAlgorithm());
 	}
 
+	@Test
+	void mlAppServicesPort() {
+		Exception exception = assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
Declare this as an IllegalArgumentException since that's the specific type we expect.